src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -29,6 +33,7 @@ public class AddressBook implements ReadOnlyAddressBook {
     {
         persons = new UniquePersonList();
         tutorials = new UniqueTutorialList();
+        consultations = new UniqueConsultationList();
```

Comment:
I think you should update the resetData() to set consultations so that it can read the data of consultations from the data file.